repose-aggregator/core/repose-core/src/main/java/org/openrepose/core/services/deploy/ContainerConfigurationListener.java

```diff
@@ -56,14 +56,11 @@ public synchronized void configurationUpdated(ContainerConfiguration configurati
                 if (ad.getCheckInterval() > 0) {
                     dirWatcher.updateCheckInterval(ad.getCheckInterval());
                 }
-
-                dirWatcher.updateArtifactDirectoryLocation(new File(ad.getValue()));
-
-                deploymentDirectory = new File(dd.getValue());
+                dirWatcher.updateArtifactDirectoryLocation(new File(ad.getValue().trim()));
+                deploymentDirectory = new File(dd.getValue().trim());
```

Comment:
Well we don't necessarily have a fully qualified path, do we? And sure, of course it's bad practice, but it's also technically valid in some filesystems.

I always struggle when it comes to imposing artificial constraints on our users. Especially when, on the other hand, artificial constraints may provide some convenience to most users.